README.md

```diff
@@ -11,7 +11,7 @@ It provides a set of APIs to directly measure performance and behavior of your s
 
 ## Project Status
 
-| Signal  | Status     | Project               |
+| Signal  | Status     | Work Tracking Project |
```

Comment:
I could agree on changing it to "Active Project". However, I do not feel it needs to be changed.